torch/fx/passes/split_module.py

```diff
@@ -7,6 +7,8 @@
 from torch.fx._compatibility import compatibility
 from torch.fx.graph_module import GraphModule
 from torch.fx.node import Node
+from torch.fx.experimental.symbolic_shapes import free_symbols
+import sympy
```

Comment:
hmm, perhaps this workaround would suffice?

if TYPE_CHECKING:
   import sympy